hpe_3par_kubernetes_manager.py

```diff
@@ -1534,31 +1534,22 @@ def verify_multipath(hpe3par_vlun, disk_partition):
             col = path.split()
             logging.getLogger().info("col :: %s" % col)
             logging.getLogger().info("col[2] :: col[3] :: %s,%s" % (col[2], col[3]))
-
-            if col[2] in disk_partition_temp:
-                logging.getLogger().info("col[2] :: %s " % col[2])
-                disk_partition_temp.remove(col[2])
-                # if '''col[4] != 'active' or '''(col[5] != 'ready' and col[5] != 'ghost') or col[6] != 'running':
-                if (col[5] != 'ready' and col[5] != 'ghost') or col[6] != 'running':
-                    logging.getLogger().info("col[4]:col[5]:col[6] :: %s:%s:%s " % (col[4], col[5], col[6]))
-                    multipath_failure_flag += 1
+            if col[2] in disk_partition_temp or col[3] in disk_partition_temp:
+                if col[2] in disk_partition_temp:
+                    current_index = 2
                 else:
-                    if col[5] == 'ready':
-                        partition_map['active'].append(col[2])
-                    elif col[5] == 'ghost':
-                        partition_map['ghost'].append(col[2])
-
-            if col[3] in disk_partition_temp:
-                logging.getLogger().info("col[3] :: %s " % col[3])
-                disk_partition_temp.remove(col[3])
-                if (col[6] != 'ready' and col[6] != 'ghost') or col[7] != 'running':
-                    logging.getLogger().info("col[4]:col[5]:col[6] :: %s:%s:%s " % (col[4], col[5], col[6]))
+                    current_index = 3
+                logging.getLogger().info("col[%s] in disk_partition_temp, :: %s" % (current_index, col[current_index]))
+                disk_partition_temp.remove(col[current_index])
+                # if '''col[4] != 'active' or '''(col[5] != 'ready' and col[5] != 'ghost') or col[6] != 'running':
+                if (col[current_index+3] != 'ready' and col[current_index+3] != 'ghost') or col[current_index+4] != 'running':
+                    logging.getLogger().info("col[%s]:col[%s]:col[%s] :: %s:%s:%s " % (current_index+2, current_index+3, current_index+4, col[current_index+2], col[current_index+3], col[current_index+4]))
                     multipath_failure_flag += 1
                 else:
-                    if col[6] == 'ready':
-                        partition_map['active'].append(col[3])
-                    elif col[6] == 'ghost':
-                        partition_map['ghost'].append(col[3])
+                    if col[current_index+3] == 'ready':

```

Comment:
Both the elif conditions gets executed twice. You can move the statements to above condition.